bin/alluxio

```diff
@@ -52,7 +52,7 @@ function printUsage {
   echo "GENERIC_COMMAND_OPTIONS supports:"
   echo -e "  -D<property=value>\t Use a value for a given Alluxio property"
   echo
-  # TODO(binfan): Fix help function for alluxio script
+  echo -e "  help   \t list available commands."
```

Comment:
@suninsky can you move this line up to the command usage instead of under GENERIC_COMMAND_OPTIONS supports:

Reply:
Thank you for your advice. This is really something I didn't notice. I will modify it.